tests/runtime/basic

```diff
@@ -324,3 +324,9 @@ EXPECT @b[1]: 0
 NAME has_key as a variable
 PROG BEGIN { @a[1] = 1; $b = has_key(@a, 1); $c = has_key(@a, 2); print(($b, $c)); exit(); }
 EXPECT (1, 0)
+
+NAME exit code
+PROG BEGIN { exit(69); }
+EXPECT_NONE 69
```

Comment:
What's this EXPECT_NONE for?

Reply:
at least one variation of EXPECT was required in runtime tests. modified it in a way to only require expect when no RETURN_CODE is provided.